### DISCUSSION TITLE
How paste pictures from clipboard (Windows 10) in wikijs

### DISCUSSION THREAD

**Discussion by github4soft** [requarks/wiki#6845] [Help / Questions]
Is there a smarter way to integrate pictures to wiki pages than uploading and then inserting?

We also noticed that when copying a text including pictures from Word and pasting them inti a wiki-page text an pictures are pasted but where are these pictures saved? We can't find them.

Thanks a lot for helping.

**Comment by P4mes4n** [comment]
You can't. In works sometimes in the Visual Editor and the image gets pasted as a Base64 String into the page itself, so it doesnt appear in the Asset dialogue, which is what you probably did. If you then convert the page to Markdown it gets messy as the whole Bas64 String will appear in your editor.

**Reply by github4soft** [reply]
Thank you. 
So we should avoid using images when pasting text.

**Comment by FarmDoc** [comment]
Pasting is quite Easy if you use the draw.io-Editor in the Markdown-Menu.

**Reply by github4soft** [reply]
Thank you.
Unfortunately most of my colleagues use the visual editor.

**Comment by github4soft** [comment]
Isn't there any existing (or planned) Plugin to paste or upload images directly in the visual editor?
We can't urge the users to use markup.
